.ci.yaml

```diff
@@ -1919,15 +1919,6 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: integration_ui_keyboard_resize
 
-  - name: Linux_android integration_ui_screenshot
```

Comment:
Could you remove the ownership entries as well?

Reply:
ahh, good catch, done